src/main/java/in/ashwanthkumar/gocd/github/GitHubPRBuildPlugin.java

```diff
@@ -227,7 +228,12 @@ GoPluginApiResponse handleGetLatestRevision(GoPluginApiRequest goPluginApiReques
             }
 
             // Remove all other branches from the response to ensure the next time those will be picked up by GoCD
-            branchToRevisionMap.entrySet().removeIf(entry -> !Objects.equals(entry.getKey(), newerRevision.getKey()));
+            if (provider instanceof GitHubProvider) {
```

Comment:
Looks good!

This is not related to GitHub generally, but to the migration, is that right?
Is it planned to be merged to master, or will it be hosted on a dedicated branch and get a special version like it's on the pom now?

Reply:
I was unsure about it, but if it's not too much of a hassle to release from a separate branch, I will do it like that 👍